src/neural/encoder.h

```diff
@@ -32,8 +32,11 @@
 
 namespace lczero {
 
+enum FillEmptyHistory {NO, FEN_ONLY, ALWAYS};
```

Comment:
Better `enum class` than old style `enum` as enum puts pretty generic name `NO` into the global namespace.